interface/src/app/greenelab/stats.spec.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test the hypergeometric test function in stats.js', function() {
+  beforeEach(module('greenelab.stats'));
+
+  var MathFuncts;
+
+  beforeEach(function() {
+    inject(function($injector) {
```

Comment:
By the way, the second version of beforeEach() is prefered here, because you only need to resolve `MathFuncts`. If you have more dependencies (such as >= 3), using $injector will be better, because you only need to put $injector as the function parameter (instead of putting three dependencies as parameters).